CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 
 ## [Unreleased]
+
+## RELEASE 5.0.0-beta.0 - 2020-03-03
+### Added
+- Resources Getter - Add a get_ids_from_request method to get all models IDs given a query or an ID list.
+- Resource Deletion - Users can now bulk delete records.
+
 ### Fixed
 - Has Many Relationships - Fix records count, i.e consider filters when counting.
```

Comment:
I am not sure this fix is in version `5.0.0-beta.0`, so it should be in the `[Unreleased]` section